src/Microsoft.ML.Data/Transforms/SkipTakeFilter.cs

```diff
@@ -211,7 +211,8 @@ private sealed class Cursor : LinkedRowRootCursorBase
             private long _rowsTaken;
             private bool _started;
 
-            public override long Batch {
+            public override long Batch
+            {
                 // SkipTakeFilter does not support cursor sets, so the batch number can always be zero.
                 get { return 0; }
             }
```

Comment:
maybe change it to this:
```
            // SkipTakeFilter does not support cursor sets, so the batch number can always be zero.
            public override long Batch => 0;
```

Reply:
Sure. XML commented it while I was at it...